### PR TITLE
DSMYSQL-1496 API Docs update for upgrade functionality

### PR DESCRIPTION
**Pull request by ayersek64**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

File: api-docs/api-reference/methods/post-save-instance-logs-version-accountid-instances-instanceid-action.rst [hunk 1]
```diff
@@ -112,7 +112,7 @@ Response
 
 **Example Save instance logs: JSON response**
 
-The following example shows the Resize instance volume response:
+The following example shows the Save instance logs response:
```
Comment (ayersek64) [inline api-docs/api-reference/methods/post-save-instance-logs-version-accountid-instances-instanceid-action.rst:115]: Opportunistic change to rectify incorrect description.

**Comment by ayersek64** [comment]
Hmm, I have merge power so I went ahead and merged it.
Interestingly, I apparently have merge power on this repo w/o needing approvals.
Guess we been rolling w/ the 'honor system'. <smirk>

**Comment by cyrichardson** [comment]
I got notifications that you'd create PRs because I'm watching the repo. 

But you did the perfect thing if you want a review from infodev team - put a comment in the Slack channel asking for a review and one of our team members will review it and merge it if you'd like for them to.

Thanks so much for contributing to the docs! We appreciate what you do!

**Pull request merged**